src/php/ext/grpc/php7_wrapper.h

```diff
@@ -154,6 +156,11 @@ static inline int php_grpc_zend_hash_find(HashTable *ht, char *key, int len,
   add_property_string(arg, name, context)
 #define php_grpc_add_property_stringl(res, name, str, len, b) \
   add_property_stringl(res, name, str, len)
+#define php_grpc_add_property_zval(res, name, val) do { \
+  zval tmp; \
+  tmp = *val; \
+  add_property_zval(res, name, &tmp); \
+  } while(0)
```

Comment:
Why you need a while(0) here

Reply:
Here use `do {} while` but loop `zero` time.

Reply:
What is the `statements` mean?
Is it necessary to have this `zero` time loop?

Reply:
I got it. Thanks!